Formula/fceux.rb

```diff
@@ -20,6 +19,9 @@ class Fceux < Formula
   depends_on "sdl2"
 
   def install
+    if build.stable?
+      ENV['CXXFLAGS'] = " -DPUBLIC_RELEASE=1 "
+    end
```

Comment:
```suggestion
    ENV["CXXFLAGS"] = "-DPUBLIC_RELEASE=1" if build.stable?
```

Reply:
I think that CXXFLAGS is appended to internal cmake compiler flags, I put a space to ensure that it will be treated as a separate argument. More arguments can be appended to this as well and I want it to be clear that spaces would be required in that case.

Reply:
I find the if block syntax change suggestion to be confusing. If that a normal way to do a single conditional. I googled ruby conditionals and that is what I got. I am an old fashioned programmer, what is wrong with a basic if block?

Reply:
I see. This might be better:
```
args = std_cmake_args
args << "-DPUBLIC_RELEASE=1" if build.stable?
```
then you can just replace `std_cmake_args` with `args` in your call to `cmake`.

Reply:
-D defines a CMake variable named public release, I do not believe this is passed to the compiler as a definition. Setting CXXFLAGS in the environment prior to calling CMake will do this.

Reply:
Very well. You can use my initial suggestion, but you can remove the spaces, as they're unnecessary. You're not adding anything to `CXXFLAGS`.

Reply:
Removed the spaces in my suggestion.